src/semidiscretization/semidiscretization_hyperbolic_parabolic.jl

```diff
@@ -0,0 +1,181 @@
+end
+
+function calc_error_norms(func, u, t, analyzer, semi::SemidiscretizationHyperbolicParabolic, cache_analysis)
+  @unpack mesh, equations, initial_condition, solver, cache = semi.semi_hyperbolic
+
+  calc_error_norms(func, u, t, analyzer, mesh, equations, initial_condition, solver, cache, cache_analysis)
+end
+
+
+function compute_coefficients(t, semi::SemidiscretizationHyperbolicParabolic)
+  compute_coefficients(semi.semi_hyperbolic.initial_condition, t, semi.semi_hyperbolic)
+end
+
+function compute_coefficients!(u_ode::AbstractVector, t, semi::SemidiscretizationHyperbolicParabolic)
+  compute_coefficients!(u_ode, semi.semi_hyperbolic.initial_condition, t, semi.semi_hyperbolic)
+end
+
+
+function rhs!(du_ode, u_ode, semi::SemidiscretizationHyperbolicParabolic, t)
+  @unpack du_ode_parabolic = semi.cache
+
+  # TODO: Taal decide, do we need to pass the mesh?
+  time_start = time_ns()
+
+  # Resize the storage for the parabolic time derivative in case it was changed
+  resize!(du_ode_parabolic, length(du_ode))
+
+  @timeit_debug timer() "rhs!" begin
+    # Compute hyperbolic time derivative
+    rhs!(du_ode, u_ode, semi.semi_hyperbolic, t)
+
+    # Compute parabolic time derivative
+    rhs!(du_ode_parabolic, u_ode, semi.semi_parabolic, t)
+
+    # Add parabolic update to hyperbolic time derivative
+    du_ode .+= du_ode_parabolic
+  end
+
+  runtime = time_ns() - time_start
+  put!(semi.performance_counter, runtime)
+
+  return nothing
+end
+
+
+"""
+    SemidiscretizationHyperbolicParabolicBR1(mesh, equations, initial_condition, solver::DGSEM;
+                                             source_terms=nothing,
+                                             boundary_conditions=boundary_condition_periodic)
+
+Convenience function to construct a semidiscretization of a hyperbolic-parabolic PDE with the BR1
+scheme for a DGSEM solver. The passed arguments must correspond to the hyperbolic setup, including
+the `solver`. Internally, a matching solver for the parabolic system is created and both a
+hyperbolic and a parabolic semidiscretization are created and passed to a
+`SemidiscretizationHyperbolicParabolic`.
+"""
+function SemidiscretizationHyperbolicParabolicBR1(mesh, equations, initial_condition, solver::DGSEM;
+                                                  source_terms=nothing,
+                                                  boundary_conditions=boundary_condition_periodic,
+                                                  RealT=real(solver))
+  semi_hyperbolic = SemidiscretizationHyperbolic(mesh, equations, initial_condition, solver;
+                                                 source_terms=source_terms,
+                                                 boundary_conditions=boundary_conditions,
+                                                 RealT=RealT)
+
+  solver_parabolic = DGSEM(solver.basis, flux_central, solver.volume_integral, solver.mortar)
```

Comment:
As far as I know, the common choice is to use the weak form volume integral for the parabolic terms, even if something more complex is used for the hyperbolic terms. Would it be better to use `create_solver_auxvars` here?

Reply:
There is no hard set rule for this, it is largely a matter of convention (or data convenience) which form is used. When we use LGL nodes the weak and strong form volume integrals are equivalent so we can use whichever we like. It is just one of those decisions we need to make and then be consistent with it moving forward :)

Reply:
What I meant was that we shouldn't use something like shock-capturing volume integrals here

Reply:
The question for me is: is it the "common" or the "only sensible" choice? For example, we could make it the default to use the weak formulation by add a keyword argument `surface_flux_parabolic=VolumeIntegralWeakFormulation()` that allows the user to override it.

So if there are is no difference and no proofs depend on using the strong form vs the weak form, I'll hardcode the weak form integral here as well.

> Would it be better to use create_solver_auxvars here?

To me, the "auxvars solver" describes the concept of the solver I need to create when computing parabolic problems by getting the gradients as  "auxiliary variables"  from a separate solver. Here, the shoe doesn't really fit, so for now I'd leave it as is (also given the fact that BR1 is already very solver specific).

Reply:
> What I meant was that we shouldn't use something like shock-capturing volume integrals here

This has convinced me. I made the weak for the default in 665e99f0 and will allow users to override this with `volume_integral_parabolic`.